Integrations/build.gradle

```diff
@@ -35,9 +35,7 @@ sourceSets {
         resources.srcDirs = ['src/main/resources']
     }
     test {
-        java.srcDirs = []
-        // TODO: test doesn't compile
-        //java.srcDirs = ['src/test/java']
+        java.srcDirs = ['src/test/java']
         resources.srcDirs = ['test_python', 'src/test/resources']
```

Comment:
Have you confirmed that the tests run without the one extra line from Colin?

Reply:
One extra line from Colin?

Reply:
restoring the java.srcDirs is necessary to tell the java plugin to compile files in that dir - i am not at all sure we need test_python in resources.srcDirs though.

The later line (to be added in an upcoming commit?) that changes "test" to "check" is a convenience to be able to easily just run java tests, instead of re-running python tests as part of java tests. With it change to "check", the python tests will still run as part of the build, you can confirm this by checking the "check" ci task output.

Reply:
In other words, the tests should run either way with the other change, this just makes it a little nicer to just run java tests for debugging and such.